src/core/level.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * This module updates the title and shortName
```

Comment:
Let's wait for @sebastian-meyer to respond... but my feeling is that this should be a "w3c/" plugin, not a "core/" plugin.

Reply:
I agree!